CONTRIBUTING.md

```diff
@@ -103,19 +111,20 @@ CSS
 .footer > p {
     color: var(--darkreader-neutral-text);
 }
-
 ```
 
 Here is a full table of available CSS variables:
 
 | Variable | Description | Use |
 |-|-|-|
-| `--darkreader-neutral-background` | Neutral background color that corresponds to the user's settings. | Mostly used for elements that have a wrong background color |
-| `--darkreader-neutral-text` | Neutral text color that corresponds to the user's settings. | Used for elements with a wrong text color |
-| `--darkreader-selection-background` | The text color setting defined by the user. | The user's Text Color setting |
-| `--darkreader-selection-text` | The background color setting defined by the user. | The user's Background Color setting |
+| `--darkreader-neutral-background` | Neutral background color corresponding to the user's settings | Mostly used for elements with a wrong background color |
```

Comment:
Not sure if changing `that corresponds` -> `corresponding` is correct grammar - More like doesn't read the sentence smoothly.